setup.py

```diff
@@ -58,9 +58,9 @@
       mpicc="mpicc"
       subprocess.check_call([mpicc,"--version"])
     os.environ["CC"]=mpicc
-    os.environ["CFLAGS"]="-w -g"
+    os.environ["CFLAGS"]="-w -g -O0"
```

Comment:
Since we have `-g -O0` help for not optimizing some local variables, especially counter like `i,j`.  Otherwise, they are optimized and we can't access their value.  We should really get rid for -`g -O0` when we distribute got make the program go faster.